packages/playground/app/Rakefile

```diff
@@ -0,0 +1 @@
+Dir.glob('lib/tasks/*.rake').each {|r| import r}
```

Comment:
Do we need rake?

Reply:
Currently we don't use it, but in the future we might start using it